src/main/java/core/basesyntax/service/operations/SupplyOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (Storage.fruits.containsKey(transaction.getFruit())
+                && transaction.getOperation() == FruitTransaction.Operation.SUPPLY) {
+            int newValue = new Storage().getQuantity(transaction.getFruit())

```

Comment:
You are creating a new instance of Storage just to get the quantity of a fruit. You should use the already existing static Storage class.

Reply:
ok, done